src/string.rs

```diff
@@ -337,6 +411,34 @@ impl String {
     Self::new_from_utf8(scope, value.as_ref(), NewStringType::Normal)
   }
 
+  // Compile-time function to create an external string resource.
+  // This resource must be assigned into a 'static mut'.
+  #[inline(always)]
+  pub const fn create_external_onebyte_const(
+    buffer: &'static [u8],
+  ) -> OneByteConst {
+    is_ascii(buffer);
+    OneByteConst {
+      vtable: &ONE_BYTE_CONST_VTABLE.delete1,
+      cached_data: buffer.as_ptr() as *const char,
+      length: buffer.len() as i32,
+    }
+  }
+
+  // Creates a v8::String from a `&'static [u8]`,
```

Comment:
Replace w/:

from a `&'static OneByteConst` which is guaranteed to be Latin-1 or ASCII.